CMakeLists.txt

```diff
@@ -200,10 +200,11 @@ else()
     set(CMAKE_VISIBILITY_INLINES_HIDDEN ON)
 endif()
 
+add_subdirectory(vendor)
+
 include(features.cmake)
 
 add_subdirectory(presets)
```

Comment:
Could also move up the presets dir, as it just contains an install directive for testing, but no build targets.

Reply:
Erring on the side of more warnings, I think I'll leave this as-is for this PR.